packages/dds/tree/src/simple-tree/toMapTree.ts

```diff
@@ -209,7 +154,7 @@ function valueToMapTree(
 function mapValueWithFallbacks(
 	// eslint-disable-next-line @rushstack/no-new-null
 	value: boolean | number | string | IFluidHandle | null,
```

Comment:
Use type `TreeValue` instead of `boolean | number | string | IFluidHandle | null`

Reply:
There are also other places in this file that could use the same change